cargo-concordium/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Add support for handling the new events about upgradeability and chain queries.
```

Comment:
Also add ability to build upgradable smart contracts.

Reply:
Also mention that run and test now allow P5 limits for execution.

And please change the version to 2.3.0. There are sufficient changes to warrant that.

Reply:
I did all this now.